onie-mk-demo.sh

```diff
@@ -144,7 +144,7 @@ tar_size="$(wc -c < "${sharch}")"
 sed -i -e "s|%%PAYLOAD_IMAGE_SIZE%%|${tar_size}|" ${output_file}
 cat $sharch >> $output_file
 echo "secure upgrade flags: SECURE_UPGRADE_MODE = $SECURE_UPGRADE_MODE, \
-SECURE_UPGRADE_DEV_SIGNING_KEY = $SECURE_UPGRADE_DEV_SIGNING_KEY, SECURE_UPGRADE_DEV_SIGNING_CERT = $SECURE_UPGRADE_DEV_SIGNING_CERT"
+SECURE_UPGRADE_SIGNING_KEY = $SECURE_UPGRADE_SIGNING_KEY, SECURE_UPGRADE_SIGNING_CERT = $SECURE_UPGRADE_SIGNING_CERT"
```

Comment:
SECURE_UPGRADE_DEV_SIGNING_KEY was not changed, it is still SECURE_UPGRADE_DEV_SIGNING_KEY. Only SECURE_UPGRADE_DEV_SIGNING_CERT was changed